common/src/main/scala/explore/GraphQLSchemas.scala

```diff
@@ -3,13 +3,39 @@
 
 package explore
 
+import scala.util.Try
+
 import clue.macros.GraphQLSchema
 import explore.model.SiderealTarget
 import explore.model.enum._
+import io.circe._
+import lucuma.core.model.Target
 
 object GraphQLSchemas {
+
   @GraphQLSchema(debug = false)
   object ObservationDB {
+    object Scalars {
+      type AsterismId    = String
+      type BigDecimal    = scala.BigDecimal
+      type DmsString     = String
+      type EpochString   = String
+      type HmsString     = String
+      type Long          = scala.Long
+      type ObservationId = String
+      type ProgramId     = String
+      type TargetId      = Target.Id
+
+      implicit val encoderTargetId: Encoder[Target.Id] =
+        Encoder.encodeString.contramap[Target.Id](id => s"t-${id.toInt}")
+      implicit val decoderTargetId: Decoder[Target.Id] = Decoder.decodeString.emapTry { str =>
+        Try(Target.Id(str.drop(2).toInt))
+      }
```

Comment:
This is temporary, as well as all Id logic. We will use `Gid` when we define its instances in `lucuma-core`.

Reply:
Disregard comment above, already migrated to `Gid`.